javascript/net/grpc/web/generator/grpc_generator.cc

```diff
@@ -1712,6 +1735,10 @@ class GrpcCodeGenerator : public CodeGenerator {
 }  // namespace grpc
 
 int main(int argc, char* argv[]) {
+  if (argc == 2 && std::string(argv[1]) == "--version") {
+    std::cout << argv[0] << " " << grpc::web::GetVersion() << std::endl;
+    return 0;
+  }
```

Comment:
nit: could you add a newline after this early return? Thanks :)

Reply:
Do you mean like this:
```c++
    return 0;
  }

  grpc::web::GrpcCodeGenerator generator;
```
Or:
```c++
    return 0;

  }
  grpc::web::GrpcCodeGenerator generator;
```

Reply:
Ahh i meant the former.. it's common in our style to have that spacing after early returns, for readability :) thanks!